action.yml

```
@@ -2,14 +2,26 @@ name: "CLI for Microsoft 365 Login"
 description: "Login to tenant using CLI for Microsoft 365"
 inputs:
   ADMIN_USERNAME:
-    description: "Username (upn) of the admin"
-    required: true
+    description: "Username (email address) of the admin"
```

Let's change this back to what is was as UPN is more appropriate than email address for authentication.

```suggestion
    description: "Username (upn) of the admin"
```